.travis_start_db.sh

```diff
@@ -4,7 +4,7 @@ set -e
 case ${DB} in
 
 scylla)
-    sudo curl -o /etc/apt/sources.list.d/scylla.list -L http://repositories.scylladb.com/scylla/repo/20fc70b18261bf832cf8e0733a27979c/ubuntu/scylladb-2.1-trusty.list
+    sudo curl -o /etc/apt/sources.list.d/scylla.list -L http://repositories.scylladb.com/scylla/repo/20fc70b18261bf832cf8e0733a27979c/ubuntu/scylladb-3.0-trusty.list
```

Comment:
We can piggyback that it could go as a separate PR.

Reply:
Ok. Putting up another PR.